server/src/main/java/org/apache/cassandra/sidecar/coordination/TokenRingProvider.java

```diff
@@ -68,7 +67,7 @@ public TokenRingProvider(InstancesMetadata instancesMetadata, InstanceMetadataFe
      * @param dc          data center
      * @return map of token ranges per Cassandra instance IP
      */
-    protected abstract Map<String, List<Range<BigInteger>>> getAllTokenRanges(Partitioner partitioner, @Nullable String dc);
+    protected abstract Map<String, List<TokenRange>> getAllTokenRanges(Partitioner partitioner, @Nullable String dc);
```

Comment:
Wondering why didn't we use `TokenRange` earlier, seems like we have that class available from before